plone/app/discussion/browser/moderation.pt

```diff
@@ -103,8 +104,14 @@
                                     <input type="hidden" name="selected_obj_paths:list" value="#"
                                            tal:attributes="value item/getURL" />
                                 </td>
-                                <td tal:content="python:item.author_name or item.Creator" />
-                                <td tal:content="python:toLocalizedTime(item.ModificationDate, long_format=1)" />
+                                <td>
+                                    <span tal:content="python:item.author_name or item.Creator">Name</span>
+                                    <tal:email tal:condition="email"><br /><a 
+                                        tal:attributes="href string:mailto:$email"
+                                        tal:content="email">Email</a></tal:email>
+                                </td>
+                                <td tal:content="python:toLocalizedTime(item.ModificationDate, 
+                                      long_format=1)" />
```

Comment:
with the rule I asked above on the ``tal:define`` in mind I though that this was another tal:define, but then just when I was about to ask you to also align ``long_format`` with ``python:`` I realized that it is a single expression that you broke into two lines.

Would you mind keeping it in a single line? Even if it's longer than 80 characters, a single line for a single statement is more readable, specially when embedding python calls inside another markup (HTML).